net/tcp/tcp.h

```diff
@@ -71,7 +71,7 @@
 #  define TCP_WBPKTLEN(wrb)          ((wrb)->wb_iob->io_pktlen)
 #  define TCP_WBSENT(wrb)            ((wrb)->wb_sent)
 #  define TCP_WBNRTX(wrb)            ((wrb)->wb_nrtx)
-#ifdef CONFIG_NET_TCP_FAST_RETRANSMIT
+#if defined(CONFIG_NET_TCP_FAST_RETRANSMIT) && !defined(CONFIG_NET_TCP_CC_NEWRENO)
 #  define TCP_WBNACK(wrb)            ((wrb)->wb_nack)
```

Comment:
Should we reuse the nack from connection instance and remove this member?

Reply:
Given the independence of the Fast Retransmit and NewReno functions, the wb_nack needs to be temporarily reserved for use when only Fast Retransmit is enabled.